lib/open_invoice/entities/payment.rb

```diff
@@ -2,7 +2,7 @@ module OpenInvoice
   module Entities
     class Payment < Base
       def index(opts = {})
-        request(:get, "/supplier/#{supplier_uuid}/payment", opts)
+        request(:get, "/v1/supplier/#{supplier_uuid}/payment", opts)
       end
     end
   end
```

Comment:
There were no updates in the docs regrading the `payment` API... so I am not sure if this needs to be updated? Wanted to check though.